src/Kdyby/Doctrine/EntityRepository.php

```diff
@@ -74,11 +74,13 @@ public function findOneBy(array $criteria, array $orderBy = null)
 
 	public function countBy(array $criteria = array())
 	{
-		return (int) $this->createQueryBuilder('e')
+		$result = $this->createQueryBuilder('e')
 			->whereCriteria($criteria)
 			->select('COUNT(e)')
 			->setMaxResults(1)
-			->getQuery()->getSingleScalarResult();
+			->getQuery()->getSingleResult();
+
+		return (int) $result[1];
```

Comment:
@josefzajac I know, I just think it's better to rely on explicit name and not on magic offset, because I was supprised it's `1` and not `0`